### DISCUSSION TITLE
can find twitch stream with no ad from web,but cant in cmd

### DISCUSSION THREAD

**Discussion by zwr-123** [streamlink/streamlink#5800] [Q&A]
### Description

i login twitch in chrome and no any sub . then i use streamlink xxxx.m3u8(the url from chrome’s network ).... , then i can enjoy any stream with no ad。
How can I get stream with no ad directly in cmd? I tried to add some headers and cookies ,but useless。

**Comment by bastimeyer** [comment]
From the Streamlink Matrix channel a couple of days ago:

https://matrix.to/#/!jBcrbbEtczLsJfiLLR:gitter.im/$bUcI476h8U1D2J6ApaUToguM3xH6zCjfOiR8FhwfE3s?via=gitter.im&via=matrix.org&via=matrix.freyachat.eu

> There are many differences between your web browser and what Streamlink does (by default) when acquiring the HLS playlists.
First, you can be signed into your Twitch user account, so unless you set the Authorization HTTP header, auth data is missing. Then there's the [playerType parameter, which Streamlink sets to `embed`](https://github.com/streamlink/streamlink/blob/6.5.1/src/streamlink/plugins/twitch.py#L258). This is set to `site` on Twitch's website. The embed value is responsible for ads being replaced with the placeholder screen. The website also always acquires a client-integrity token, which Streamlink will only acquire when the access token request failed without having a client-integrity token. And in addition to all that, the HTTP GET parameters are slightly different on the usher.ttvnw.net endpoint where the HLS playlists are requested, compared to their website. This shouldn't matter though, as the access token and its signature is what's important here.

**Reply by zwr-123** [reply]
i am noob in code,so What can I do in cmd. This is my command:
streamlink  --http-header "Authorization=OAuth xxx"   --http-header "Client-Integrity=xxxx" 
What else do I need to add
